src/core/translation/locales/translation.ts

```diff
@@ -0,0 +1,12 @@
+import languages from './languages.json';
+import skills from './skills.json';
+import socialCauses from './socialCauses.json';
+
+export function generateTranslationFile(theLanguage) {
+  const translationFile = {};
+  const combinedTranslations = Object.assign({}, skills, languages, socialCauses);
+  for (const key in combinedTranslations) {
+    translationFile[key] = combinedTranslations[key][theLanguage];
+  }
+  return translationFile;
+}
```

Comment:
These changes are OK and the PR can be merged, but what's the purpose of this PR? I think just translating SKILLS and SOCIAL_CAUSES to another language won't help us that much. If we are going to translate all pages, we have to define variables page by page.

Reply:
@chouske 

Marjan is right. We need to define variables for each page. And all the variables should be in the i18n file with correspending translations.